docs/modules/installation/pages/installation.adoc

```diff
@@ -79,6 +88,15 @@ Click on the link in the output file to log in.
 
 See the following topics might be useful after installation:
 
-* xref:configuration:configuration.adoc[Manual configuration changes]
-* xref:troubleshooting:debugging.adoc[Troubleshooting]
-* xref:reference:reference.adoc[Useful tools and commands]
\ No newline at end of file
+* xref:configuration:configuration.adoc[]
+* xref:troubleshooting:debugging.adoc[]
+* xref:reference:reference.adoc[]
+
+== Updating OpenNMS service settings
+
+If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by doing an upgrade of the helm chart.
```

Comment:
```suggestion
If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by upgrading the Helm chart.
```

Reply:
Tweaked the text to match closer to Helm's terminology.
```suggestion
If you make changes to the Helm chart definition, such as modifying `values.yaml`, updating one of the init scripts, or changing a container image version, you can redeploy the container by updating the Helm release.
```